packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## v2.2.0-dev (Nov 3, 2023)
+
 - (A11y) Ensure active user interface components have sufficient contrast [#1534](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1534)
 - (A11y) Ensure form fields and icons have accessible labels [#1526](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1526)
+- (A11y) Ensure all interactive functionality is operable with the keyboard [#1546]https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1546
```

Comment:
Does this comment cover both changes in this PR? 

It's ok to reference the same PR with 2 line items. For example you could do this:

- Fix ally stuff [#1546]
- Implement gif option for basket [#1546]

Reply:
Or mention both fixes in one line. But reading it, it sounds like you missed the gift option stuff.